src/main/java/webApp/webController.java

```diff
@@ -33,6 +34,8 @@ public Response createSurvey(@RequestBody SurveyMessage surveyMessage) {
         for (QuestionMessage question : surveyMessage.getQuestions()) {
             if (question.getType().equals("openEnded")) {
                 questionList.add(new OpenEndedQuestion(question.getQuestion()));
+            } else if(question.getType().equals("numberQuestion")) {
```

Comment:
can we create "numberQuestion" as a constant, seems to be used very often in this class

Reply:
I think this needs to be a future issue, based on comment section above